OnePassword.NET/OnePasswordManager.Items.cs

```diff
@@ -71,12 +71,12 @@ public Item SearchForItem(IItem item, IVault? vault = null, bool? includeArchive
     /// <inheritdoc />
     public Item CreateItem(Template template, IVault vault)
     {
-        if (vault.Id.Length == 0)
+        if (vault == null || vault.Id.Length == 0)
```

Comment:
```suggestion
        if (vault is null || vault.Id.Length == 0)
```
Prefer pattern matching.